web/views/pages/index.ejs

```diff
@@ -99,8 +99,8 @@
         <script defer src="<%= resolvePath('dist/vendor.' + vendorHash + '.js') %>"></script>
         <script defer src="<%= resolvePath('dist/bundle.' + bundleHash + '.js') %>"></script>
 
-        <!-- Google Analytics -->
-        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-79729918-1"></script>
+        <!-- Plausible -->
+        <script defer data-domain="app.electricitymap.org" src="https://plausible.io/js/plausible.local.js"></script>
```

Comment:
Probably shouldn't still be local? :)

Reply:
Added this comment about testing on mobile apps @Kongkille

Reply:
Ah great point. We should test if the app can send on localhost